R/position-nudge2.R

```diff
@@ -1,15 +1,19 @@
+# COPIED FROM 'ggrepel' unchanged except for being exported
+# This is only temporary patch until 'ggrepel' is updated
```

Comment:
Could I ask what this means? I don't think I understand.

Reply:
Sorry! It is now deleted. (At first I had added the files to my own package... and later copied them back.)